xml/System.Net.Http/HttpClient.xml

```diff
@@ -661,7 +661,7 @@ This property has no effect on any of the <xref:System.Net.Http.HttpClient.Send%
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response (including content) is read.  
+ This operation will not block. The returned <xref:System.Threading.Tasks.Task%601> object will complete after the whole response (including content) is read. The behavior is the same as if <xref:System.Net.Http.HttpCompletionOption.ResponseContentRead> has been explicitly specified.
```

Comment:
Only `GetAsync()` takes `HttpCompletionOption` as an arg, mentioning a value that cannot be actually passed to this method sounds somewhat weird. Or we don't care?

Reply:
Yeah, that makes sense, I checked only few and assumed there are overloads for all methods. good catch!